core/store/migrate/migrations/0195_add_not_null_to_evm_chain_id_in_job_specs.go

```diff
@@ -36,6 +41,25 @@ const (
 
 // nolint
 func Up195(ctx context.Context, tx *sql.Tx) error {
+	chainID, set := os.LookupEnv(EVMChainIDNotNullMigrationHelper)
+	if set {
+		updateQueries := []string{
+			`UPDATE direct_request_specs SET evm_chain_id = $1 WHERE evm_chain_id IS NULL;`,
+			`UPDATE flux_monitor_specs SET evm_chain_id = $1 WHERE evm_chain_id IS NULL;`,
+			`UPDATE ocr_oracle_specs SET evm_chain_id = $1 WHERE evm_chain_id IS NULL;`,
+			`UPDATE keeper_specs SET evm_chain_id = $1 WHERE evm_chain_id IS NULL;`,
+			`UPDATE vrf_specs SET evm_chain_id = $1 WHERE evm_chain_id IS NULL;`,
+			`UPDATE blockhash_store_specs SET evm_chain_id = $1 WHERE evm_chain_id IS NULL;`,
+			`UPDATE block_header_feeder_specs SET evm_chain_id = $1 WHERE evm_chain_id IS NULL;`,
+		}
+		for i := range updateQueries {
+			_, err := tx.Exec(updateQueries[i], chainID)
+			if err != nil {
+				return errors.Wrap(err, "failed to set missing evm chain ids")
+			}
+		}
+	}
+
 	_, err := tx.ExecContext(ctx, addNullConstraintsToSpecs)
 	return errors.Wrap(err, "failed to add null constraints")
```

Comment:
Can the error include a message about which env var to set in order to have a successful re-run?

Reply:
Well the success of the migration doesn't depend on if the env var is set or not, it should work without the env var set for eg. when node isn't running on EVM.

Reply:
I thought the point of the env var was an escape hatch to be able to manually set the value?

Reply:
Oh I think I see now :thinking: 

But what if migrations are invoked manually? I think there is still a case where they won't understand that the env var is available to them.

Reply:
The issue is that we can't really discern when to force the NOP to provide the evmChainID env var, because we don't have info about if EVM is enabled in the migration. 
So actually this PR should give us a permanent function that knows when to inject env vars into migrations. Instead of somehow logging info that you should provide a env var.

Reply:
@jmank88 My assumption is that it is best

> Oh I think I see now 🤔
> 
> But what if migrations are invoked manually? I think there is still a case where they won't understand that the env var is available to them.

My assumption is that its best to just do this automatically by setting the env vars in the code. If by invoking manually you mean running db migrate command, the same thing would happen.

Reply:
Oh I didn't realize that we were already passing full config to the `db` sub-commands since they are underneath `node` :thinking: No problem then I guess.